meson.build

```diff
@@ -60,17 +60,12 @@ else
 endif
 
 
-#gtest_proj = subproject('gtest')
-#gtest_dep = gtest_proj.get_variable('gtest_dep')
-#gtest_dep = dependency('gtest', main: true)
-#gmock_dep = gtest_proj.get_variable('gmock_dep')
+# Unit tests
 
-#test_src = ['test/example.cpp', 'tests/gtest_all.cc']
+if not meson.is_cross_build()
+    gtest_proj = subproject('gtest')
```

Comment:
You never actually use this variable. :)

Force using `subproject('gtest')` means that even if there is a gtest_main installed on the system, the subproject will override it. Meson is supposed to try both, so deleting this line "should" still work.

(You can force dependencies to be downloaded using subprojects by passing the option `--wrap-mode=forcefallback` to the meson invocation.)